lcservice-go/service/command.go

```diff
@@ -0,0 +1,22 @@
+package service
+
+type CommandsDescriptor struct {
+	Descriptors []commandDescriptor `json:"commands"`
+}
+
+var commandAllowedRequestParamTypes = map[string]struct{}{
+	RequestParamTypeString: {},
+	RequestParamTypeEnum:   {},
+	RequestParamTypeInt:    {},
+	RequestParamTypeBool:   {},
+	RequestParamTypeFlag:   {},
+}
+
+type CommandNamespace = string
+type CommandName = string
+type commandDescriptor struct {
+	Namespace CommandNamespace `json:"namespace"`
```

Comment:
Will we know the service name when fetching command usage from redis?

Reply:
Yes, the usage will be keyed on serviceName, and the SH receiving the metadata from the Service also knows the name, so it will be able to enforce it rather than taking it on face value from the Service.